docs/reference/search/request/from-size.asciidoc

```diff
@@ -1,29 +1,45 @@
-[[request-body-search-from-size]]
-==== From / Size
+[discrete]
+[[paginate-search-results]]
+=== Paginate search results
 
-Pagination of results can be done by using the `from` and `size`
-parameters. The `from` parameter defines the offset from the first
-result you want to fetch. The `size` parameter allows you to configure
-the maximum amount of hits to be returned.
+By default, the search API returns documents for only the top 10 search
+results, starting with the first, most relevant document.
 
-Though `from` and `size` can be set as request parameters, they can also
-be set within the search body. `from` defaults to `0`, and `size`
-defaults to `10`.
+To paginate a larger set of search results, you can use the search API's `size`
+and `from` parameters. The `from` parameter is a zero-indexed offset from
+the first result you want to fetch. The `size` parameter sets the maximum
+number of results to return.
+
+[IMPORTANT]
+====
+The combined document count, including offset and and returned documents, for
+the `from` and `size` parameters cannot exceed the
+<<index-max-result-window,`index.max_result_window`>> index setting, which
+defaults to 10,000 documents.
+====
```

Comment:
I think the point here is that by default, you can page through a maximum of 10,000 documents. I'd say that, and then xref the settings, but probably not as a note. Instead,
I'd pull some of the cautionary info about deep paging into an important note from the definitive guide.